content/300-guides/300-upgrade-guides/300-upgrading-to-use-preview-features/050-enabling-named-constraints.mdx

```diff
@@ -118,121 +113,89 @@ Use the following guide if your project uses Prisma Migrate. Choose the upgrade
     }
     ```
 
-1.  Optionally introspect your **production database** and double-check that constraint names in production and development are in sync. If you have external applications or scripts that rely on a specific constraint name in production, make sure that these constraint names are represented in the Prisma schema.
+### Upgrade path 2: I want to use Prismas default constraint and index names
 
-    In the following example, the `Post` model has a foreign key constraint with a custom name in production that does not match development:
+If you want to keep the clean look of your Prisma schema without explicitly spelled out constraint and index names and have no reasons preventing you from renaming their names in the database you can create a migration to update the names. With the preview flag enabled, run `prisma migrate dev` to create a migration updating the constraint names to Prisma's defaults. Afterwards, do not forget to `prisma migrated deploy` against your production environment if you have one to also update the names there. The schema below has no explicit constraint or index names spelled out, so Prisma will infer them.
```

Comment:
Split this section into paragraphs

If you want to keep the clean look of your Prisma schema without explicitly spelled out constraint and index names and have no reasons preventing you from renaming their names in the database you can create a migration to update the names. 

With the preview flag enabled, run `prisma migrate dev` to create a migration updating the constraint names to Prisma's defaults.

Afterwards, do not forget to `prisma migrated deploy` against your production environment if you have one to also update the names there. The schema below has no explicit constraint or index names spelled out, so Prisma will infer them.

Reply:
Done